lib/nudge_web/controllers/site_controller.ex

```diff
@@ -2,8 +2,15 @@ defmodule NudgeWeb.SiteController do
   use NudgeWeb, :controller
 
   def index(conn, _params) do
-    sites = Nudge.Accounts.list_sites()
-    render(conn, "index.html", sites: sites)
+    site = Nudge.Accounts.list_sites()
+    current_user_id = Nudge.Accounts.get_site!(site.id)
+
+    user_site =
+      Site
+      |> Nudge.Accounts.list_user_sites(current_user_id)
+      |> Repo.all()
```

Comment:
@Menubatenny Not clear what this would produce. Variable names are a bit misleading here.

is this `site = Nudge.Accounts.list_sites()` needed? 

does `current_user_id = Nudge.Accounts.get_site!(site.id)` this give us the user-id ?

shouldn't the user id be taken from current_user session?